libraries/helpers.rb

```diff
@@ -0,0 +1,18 @@
+module HabitatCookbook
+  module Helpers
+    def hab(*command)
+      begin
+        if defined?(shell_out_with_timeout!)
```

Comment:
And to reiterate -- consider what happens when i push `shell_out_with_timeout!` up into the top level Chef::Mixin::ShellOut module in order to make it broadly available to all providers (since we need it outside of the package providers and someone submitted a PR to try to wrap service providers (i think) with timeouts that needs to get refactored to use shared code instead of reinventing a different solution to the same problem...)